project/SavedLogs.scala

```diff
@@ -0,0 +1,62 @@
+package scala.build
+
+import java.io.StringWriter
+
+import scala.collection.mutable
+import sbt._
+import Keys._
+import org.apache.logging.log4j.core
+import org.apache.logging.log4j.core.appender.{AbstractAppender, WriterAppender}
+import sbt.internal.util.StringEvent
+
+/** Save MiMa logs so they don't get lost in lots of debug output */
+object SavedLogs {
```

Comment:
Should we rename this (and the file) seeing as it's MiMa specific?

Reply:
I used a more general name in case we want to apply it to other tasks later.

Reply:
I find it misleading as it is.

Reply:
I share the sentiment, but we can always change this since it's nothing public